.github/workflows/release.yml

```diff
@@ -28,17 +29,19 @@ jobs:
           token: ${{ secrets.GH_TOKEN }}
   publish:
     name: Publish package
-    runs-on: macos-11
+    runs-on:
+      group: Default
```

Comment:
Shouldn't it be "Default Larger Runners" ?

Reply:
macOS is in different group